tests/special_coverage/test_spec_cov_views.py

```diff
@@ -45,6 +45,11 @@ def test_special_coverage_view(self):
         self.assertEqual(response.context['special_coverage'], sc)
         self.assertEqual(response.context['content_list'].count(), sc.get_content().count())
         self.assertEqual(response.context['content_list'][0].id, content.id)
+        self.assertEqual(response.context['recirc'], None)
```

Comment:
Hm, yeah, I'd say add some content to it and then just assert that `recirc` doesn't have any unpublished data and that it doesn't have itself.